### PR TITLE
Added a couple of sentences from the Presto JDBC Driver section to the jdbc.rst file

### PR DESCRIPTION
**Pull request by m57lyra**
Used text from the Presto: The Definitive Guide section on the Presto JDBC Driver section on page 30 for the installation/jdbc page as per the interview exercise requirements.

Screen cap includes local git activity as well.
Screen Shot 2020-07-19 at 4 59 11 PM

@mosabua

### CODE REVIEW

**Comment by findepi** [comment]
@mosabua @mattsfuller can the wording be used as is?

**Comment by mosabua** [comment]
@findepi .. this is just an exercise. We will update and get it to merge later.

File: presto-docs/src/main/sphinx/installation/jdbc.rst [hunk 1]
```diff
@@ -3,6 +3,11 @@ JDBC Driver
 ===========
 
 Presto can be accessed from Java using the JDBC driver.
+JDBC is a standard API that provides the necessary methods such as querying,
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/installation/jdbc.rst:6]: new paragraph (empty line).

File: presto-docs/src/main/sphinx/installation/jdbc.rst [hunk 2]
```diff
@@ -3,6 +3,11 @@ JDBC Driver
 ===========
 
 Presto can be accessed from Java using the JDBC driver.
+JDBC is a standard API that provides the necessary methods such as querying, 
+inserting, deleting, and updating data in a relational database. Many client and server-side 
+applications running on the JVM implement features for database management, reporting, and 
+other aspects and are using JDBC to access the underlying database. All of these applications 
+can use Presto with the JDBC driver.
 Download :maven_download:`jdbc` and add it to the class path of your Java application.
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/installation/jdbc.rst:11]: new paragraph (empty line).

**Comment by m57lyra** [comment]
Need to reword text.

**Pull request closed without merging**